Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -31,11 +76,22 @@ void OnSelect()
                 if (placing)
                 {
                     SpatialMappingManager.Instance.DrawVisualMeshes = true;
+
+                    Debug.Log(gameObject.name + " : " + "Removing existing world anchor if any.");
+
+                    // Remove existing world anchor when moving an object.
+                    DestroyImmediate(gameObject.GetComponent<WorldAnchor>());
+
+                    // Delete existing world anchor from anchor store when moving an object.                    
+                    anchorStore.Delete(SavedAnchorFriendlyName);
```

Comment:
Sure thing. Will add that. 
However, if you were trying to solve some race condition between anchor store being ready and user tapping that wont affect that. The likelihood is low though from my observations.